src/java.base/share/classes/javax/crypto/CipherInputStream.java

```diff
@@ -156,8 +154,6 @@ private int getMoreData() throws IOException {
         ensureCapacity(readin);
         try {
             ofinish = cipher.update(ibuffer, 0, readin, obuffer, ostart);
-        } catch (IllegalStateException e) {
```

Comment:
This is another one of those I would probably leave alone, just so it's clear what should be done.

Reply:
Keeping the original.